packer/playbook-node.yaml

```diff
@@ -0,0 +1,22 @@
+- name: Kubernetes node preparation
+  hosts: all
+  become: true
+  vars:
+    kubernetes_repo_distribution: 'xenial'
+    kubernetescni_version: '0.7.5'
+    docker_version: '18.06.3~ce~3-0~ubuntu'
+  roles:
+    - ../../roles/kube-node-common
+
+- name: Installing and configuring furyagent
```

Comment:
Can we leverage the furyagent role for this play? This way, we have all the furyagent-related logic in the same place.

Reply:
Done!